docker-compose-dev.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  web:
+    build: ./
+    volumes:
+      - ./src/presence_analyzer:/src
```

Comment:
This is broken. Have you checked if the app auto-reloads in dev mode? Because if you did, you would see it doesn't ;)